cli.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+'use strict';
+
+
+var program = require('commander'), fs = require('fs'), nconf = require('nconf'), fileview = require('./index');
```

Comment:
Each declaration should be done on every line.

Makes the code more readable.

I will advice you install [eslint](eslint.org) and lint with at least [airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb). Will help improve coding pattern.